build-tools/packages/build-tools/README.md

```diff
@@ -157,12 +157,16 @@ For example:
 {
 	"fluidBuild": {
 		"tasks": {
+         "build": ["...", "build:docs", "copy:docs"]
```

Comment:
If you change the markdown code block type to json5 it should make the comments render properly in GitHub.

Reply:
No, it is because I used space instead of tabs.  I have updated it.